cmake/Modules/Packages/LATTE.cmake

```diff
@@ -19,6 +19,14 @@ if(DOWNLOAD_LATTE)
   set(LATTE_MD5 "820e73a457ced178c08c71389a385de7" CACHE STRING "MD5 checksum of LATTE tarball")
   mark_as_advanced(LATTE_URL)
   mark_as_advanced(LATTE_MD5)
+
+  # CMake cannot pass BLAS or LAPACK library variable to external project if they are a list
+  list(LENGTH BLAS_LIBRARIES} NUM_BLAS)
+  list(LENGTH LAPACK_LIBRARIES NUM_LAPACK)
+  if((NUM_BLAS GREATER 1) OR (NUM_LAPACK GREATER 1))
+    message(FATAL_ERROR "Cannot compile downloaded LATTE library due to a technical limitation")
```

Comment:
Not a super useful error message.

Reply:
Suggest a better one.

Reply:
That is just as confusing because this is what this is attempting to do. Since this is only a stopgap measure it is still significantly better than getting an error about not finding the command "-dl" or something similarly random. At least people will than know who and what to ask and an explanation can be provided. The hope is that this will affect only a small number of people. It wasn't a problem on Fedora until it introduced "flexiblas".

Reply:
My problem is that the error message that no real explaination what is wrong, maybe just adding a link to https://github.com/lammps/lammps/issues/2964 would help.

Reply:
I cannot think of a one line explanation. Otherwise it would be there.